src/formFieldAppender.test.js

```diff
@@ -0,0 +1,74 @@
+jest.mock('DEGJS/domUtils');
+import polyfillsForTests from './__mocks__/pollyfills';
+import formFieldAppender from './formFieldAppender';
+
+polyfillsForTests();
+const classes = {
+    item: 'js-ffa-item'
+}
+let removeBtn;
+let wrapperEl;
+
+describe('formFieldAppender', () => {
+    beforeEach(() => {
+        document.body.innerHTML = `
+            <div class="example__interface js-example-1">
+                <div class="${classes.item}">
+                    <div class="field">
+                        <label for="phone1">Phone number</label>
+                        <input name="phone1" id="phone1" type="text">
+                    </div>
+                    <button class="js-ffa-add-trigger">Add</button>
+                    <button class="js-ffa-remove-trigger">Remove</button>
+                </div>
+            </div>
+        `;
+        wrapperEl = document.querySelector('.js-example-1');
+        removeBtn = document.querySelector('.js-ffa-remove-trigger');
+    });
+
+    it('should remove item', () => {
```

Comment:
This is the exact same test as the "should remove item if callback is true" test below. Maybe you could omit the shouldRemoveItemCallback stuff from this test, which would then test to make sure that the item is removed when no callback is provided.